Library/.rubocop.yml

```diff
@@ -342,12 +341,6 @@ Style/BlockDelimiters:
     - "sig"
   Exclude:
     - "Homebrew/**/*_spec.rb"
-    - "Homebrew/**/shared_examples/**/*.rb"
-
-# TODO: remove this when possible.
-Style/ClassVars:
```

Comment:
Hmmm, turns out this _does_ still reveal bad things, but only if I run `brew style $(brew --repo homebrew/cask)`. I'll add some in-line disables in a PR in Cask, since I can because the filepaths for the `Style/DisableCopsWithinSourceCodeDirective` are only `*.rb` and these scripts don't have file extensions. 😏